### ISSUE TITLE
iPhone X Log in page layout bugged

### ISSUE THREAD

**Issue by Raalse** [Inter-Actief/amelie#318]
The layout is really hard to follow. The buttons to go to the log in or cancel are so small they're basically impossible to press. Thanks!
<img width="490" alt="screen shot 2018-02-22 at 20 22 28" src="https://user-images.githubusercontent.com/23061550/36559410-2c8e35fa-180e-11e8-9e73-3c1bfe720ccc.png">

**Comment by SebastiaandenBoer** [comment]
This is an issue on most devices with a small viewport width. We should think about redesigning this page to look good on any device. Also, should this page really have the full menu instead of a simple back/ home button?

**Comment by windwarrior** [comment]
What I don't get from the screenshot tho, is that the webpage doesn't stretch to the sides of the viewport. When manually selecting iPhone X in the chrome dev tools the website appears better.

**Comment by SebastiaandenBoer** [comment]
Could be that this is because it is used in a frame instead of Safari.

**Comment by SebastiaandenBoer** [comment]
I discovered that the website is only optimized for a width of 320px or larger. This render seems to be lower than that, around 250px.

**Comment by SebastiaandenBoer** [comment]
Supposedly, adding a min-width 320px; in the CSS solves this problem, but could not be tested due to a lack of testing devices.
